CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @digitalbazaar/ed25519-signature-2020 Changelog
 
+## 6.0.0 - 2023-TBD
+
+### Changed
+- **BREAKING**: Remove support for `expansionMap` from `matchProof()`.
+  (Removed in `jsonld-signatures@11`.)
```

Comment:
```suggestion
- Remove unused `expansionMap` from `matchProof()` as it was removed
  from `jsonld-signatures@11` which is required since version `5.0`.
```

Reply:
Updated here [75c4354](https://github.com/digitalbazaar/ed25519-signature-2020/pull/26/commits/75c43540647f1461ae61e0e3d5277fd39dd7ca98)